modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc

```diff
@@ -30,9 +30,31 @@ Your application runs a replicator (also referred to here as a client), which  w
 
 Subsequent sections provide additional details and examples for the main configuration options.
 
+== Replication Concepts
 
-== Replication Protocol
+Couchbase Lite allows for one database for each application running on the mobile device.
+This database can contain one or more scopes.
+Each scope can contain one or more collections.
+
+To learn about Scopes and Collections, see xref:{param-module}:database.adoc[]
+
+You can set up a replication scheme across any of the three data levels:
+
+Database:: All scopes and collections in the database are synced,
+including the `_default` scope and the `_default` collection.
+
+Scope:: All the collections in a named scope.
```

Comment:
Not sure what this means in this context. However, CBL doesn't provide an API to specify only a scope for syncing all collections in the scope. CBL only provides API to specify a list of collections.